src/components/layout/layout.tsx

```diff
@@ -30,28 +30,34 @@ const FullHeightContainer = styled.div`
   }
 `;
 
-const Main = styled.main`
+const Main = styled.main<{ fullWidth?: boolean }>`
   /** make the element take up all available space */
   flex-grow: 1;
 
-  display: grid;
+  ${({ fullWidth }) =>
+    !fullWidth &&
```

Comment:
This looks wrong. If you remove the `display: grid`, the full width trick with `grid-column: 1 / 4;` shouldn't work anymore. Also, the main component now needs to know if there is a full width children, which shouldn't be the case.